python/cuml/test/test_fil.py

```diff
@@ -301,7 +302,7 @@ def test_fil_skl_regression(n_rows, n_columns, n_estimators, max_depth,
     fil_preds = np.asarray(fm.predict(X_validation))
     fil_mse = mean_squared_error(y_validation, fil_preds)
 
-    assert fil_mse == pytest.approx(skl_mse, 1e-5)
+    assert fil_mse == pytest.approx(skl_mse, 1e-4)
```

Comment:
I think 1e-4 is a fine margin, but why the change? Preventive?

Reply:
I ran a bunch of tests and found that there is occasional tiny variance in the accuracy. Usually we are fine within 1e-5 but once in a while it exceeds that by a little and I don't want to leave a lurking transient failure.